readme.md

```diff
@@ -158,37 +159,47 @@ environment, set the same environment variables there using the method appropria
 
 Once Heroku restarts your application or your Docker container is refreshed, you should have persistent media storage!
 
-### Sending email from the contact form
+# Next steps
 
-The following setting in `base.py` and `production.py` ensures that live email is not sent by the demo contact form.
+Hopefully after you've experimented with the demo you'll want to create your own site. To do that you'll want to run the `wagtail start` command in your environment of choice. You can find more information in the [getting started Wagtail CMS docs](http://wagtail.readthedocs.io/en/latest/getting_started/index.html).
 
-`EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'`
 
-In production on your own site, you'll need to change this to:
+# Contributing
 
-`EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'`
+If you're a Python or Django developer, fork the repo and get stuck in! If you'd like to get involved you may find our [contributing guidelines](https://github.com/wagtail/bakerydemo/blob/master/contributing.md) a useful read.
 
-and configure [SMTP settings](https://docs.djangoproject.com/en/1.10/topics/email/#smtp-backend) appropriate for your email provider.
+### Preparing this archive for distribution
 
-### Ownership of demo content
+If you change content or images in this repo and need to prepare a new fixture file for export, do the following on a branch:
 
-Most of the images and textual content in this project are sourced from Wikipedia, and is therefore open source and copyright-free.
+`./manage.py dumpdata --natural-foreign --indent 2 -e auth.permission -e contenttypes -e wagtailcore.GroupCollectionPermission -e wagtailimages.filter -e wagtailcore.pagerevision -e wagtailimages.rendition  -e sessions > bakerydemo/base/fixtures/bakerydemo.json`
 
-### Instructions for demo developers preparing this archive for distribution
+Please optimize any included images to 1200px wide with JPEG compression at 60%. Note that `media/images` is ignored in the repo by `.gitignore` but `media/original_images` is not. Wagtail's local image "renditions" are excluded in the fixture recipe above.
 
-If you change content or images in this repo and need to prepare it for export, do the following on a branch:
+Make a pull request to https://github.com/wagtail/bakerydemo
 
-```
-./manage.py dbshell
-delete from django_session;
-delete from wagtailimages_rendition;
```

Comment:
I removed these lines because the included fixtures recipe already excludes the sessions and renditions table, so this was redundant.